python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/management_security.schema.yml

```diff
@@ -13,6 +13,7 @@ keys:
         type: str
         deprecation:
           warning: true
+          removed: true
```

Comment:
as decided please move the entropy_source at the end and update the porting-guide

Reply:
Fixed the schema. Update the porting guide.